examples/sparseserver-ui/clientrequirements.txt

```diff
@@ -0,0 +1 @@
+streamlit==1.8.1
```

Comment:
this could be put in the client dockerfile

Reply:
I normally use requirements.txt incase we need more than one python library in the future, but i'll remove it and add the  installation directly to the Dockerfile